### DISCUSSION TITLE
Issue when changing the "name"

### DISCUSSION THREAD

**Discussion by jjtoard** [kotope/aio_energy_management#49] [Q&A]
I created multiple cheapest hours binary sensors.  all gave them a unique ID but made the mistake to give them the same 
name: My Cheapest Hours

Now all the binary sensors can only be found via the name_<sequence numer it looks>
I already removed the hacs integration, restarted home assistant multiple times but I'm stuck with the binary_sensors all having them 

![image](https://github.com/user-attachments/assets/5fae8b19-d1d9-45ea-a3ae-601cbbadcad5)

**Comment by kotope** [comment]
This is something how the Home Assistant entity names work.

You can 'reset' the situation by removing AIO Energy Management configuration and restart HA.
After that go to Home Assistant 'settings -> devices & services -> entities' and search for 'my_cheapest_hours'. You should be able to remove those once they are orphaned (as in no integration found for those entities anymore).

**Reply by jjtoard** [reply]
Ok thanks,  tought uninstalling the hacks integration would do the trick but apparently not.  Worked for me.
